doc/source/reference/random/extending.rst

```diff
@@ -70,9 +70,9 @@ directly from the ``_generator`` shared object, using the `BitGenerator.cffi` in
     :start-after: dlopen
 
 
-New Basic RNGs
-==============
-`~Generator` can be used with other user-provided BitGenerators. The simplest
+New Bit Generators
+==================
+`~Generator` can be used with other user-provided `~BitGenerators`. The simplest
 way to write a new BitGenerator is to examine the pyx file of one of the
```

Comment:
The `s` on `BitGenerators` will [not link](https://10736-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/random/extending.html#new-bit-generators) properly. Maybe "with any `BitGenerator`"

Reply:
Or `` `~BitGenerator`\ s``

Reply:
I used @eric-wieser's suggestion